packages/data/addon/services/navi-formatter.ts

```diff
@@ -13,7 +13,7 @@ export default class NaviFormatterService extends Service {
     const paramValues = Object.values(allParams);
 
     const name = alias || columnMetadata?.name || '--';
-    if (paramValues.length) {
+    if (paramValues.length && !alias) {
       return `${name} (${paramValues.join(',')})`;
     } else {
       return name;
```

Comment:
Can we just add something like this at the top of the function. I feel like that would be more explicit than this check
```js
if (alias) {
  return alias;
}
```

Reply:
Sure I'll try that!